README.rst

```diff
@@ -207,6 +207,13 @@ Run tests with:
 
     ./runtests.py
 
+Visualizer
+----------
+
+.. figure:: src/image/visual-topology-network.PNG
```

Comment:
Please move the image to this path: `docs/images/visualizer.png` (including lowercase PNG for consistency with the rest of the openwisp conventions)